lambda/models/state_machine/create_model.py

```diff
@@ -16,25 +16,62 @@
 
 from copy import deepcopy
 from typing import Any, Dict
+import json
+import os
+
+import boto3
+from botocore.exceptions import ClientError
 
 
 def handle_set_model_to_creating(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
     """Set DDB entry to CREATING status."""
     output_dict = deepcopy(event)
-    output_dict["create_infra"] = True
+    output_dict["create_infra"] = 'modelConfig' in event
     return output_dict
 
 
 def handle_start_copy_docker_image(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
     """Start process for copying Docker image into local AWS account."""
     output_dict = deepcopy(event)
+
+    lambdaClient = boto3.client("lambda")
```

Comment:
same note from my other comment, let's add any clients to the top of the file to take advantage of "warm" lambda functions that can reuse the clients defined at the module level